src/platform/packages/shared/kbn-unified-data-table/src/hooks/use_row_height.ts

```diff
@@ -138,10 +136,18 @@ export const useRowHeight = ({
     [configRowHeight, consumer, key, onUpdateRowHeight, storage]
   );
 
+  useEffect(() => {
```

Comment:
We can't set input value within `onChangeRowHeightLines` because sometimes it comes from URL, so this function is not triggered. If someone more experienced with Kibana has better idea, any insight is appreciated!